packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -340,5 +340,5 @@ export const Dialpad = (props: DialpadProps): JSX.Element => {
 
 const sanitizeInput = (input: string): string => {
   // remove non-valid characters from input: letters,special characters excluding +, *,#
-  return input.replace(/[^\d*#+]/g, '');
+  return input.replace(/[^\d*#+.]/g, '');
```

Comment:
The `.` is a wildcard that matches everything. This regex doesn't seem right to me

Reply:
OR are we saying that a dot is also allowed
If yes, why is a dot also allowed?

Reply:
the dot is now allowed since thats what we are appending to the string to create the `...`